request/consumers/base.py

```diff
@@ -12,18 +12,15 @@ def __init__(self, key):
         self.host = host
         self.app_base_route = ''
 
-    def make_request(self, method, query=None, body=None):
+    def make_request(self, query=None, body=None):
         try:
             connection = HTTPSConnection(self.host, context=ssl._create_unverified_context())
         except AttributeError:
             connection = HTTPSConnection(self.host)
-        params = ''
-        if query is not None:
```

Comment:
While slightly janky, this is necessary to support arbitrary params without forcing `"?cmd=shows"`. I'll add an alternative below